packages/ember-metal/tests/map_test.js

```diff
@@ -519,13 +512,6 @@ QUnit.module('OrderedSet', {
   }
 });
 
-QUnit.test('OrderedSet() without `new`', function() {
```

Comment:
Same as above, not sure why this would have to be removed in debug builds.  I believe we only strip the class call check helper method in production builds...